.coderabbit.yaml

```
@@ -0,0 +1 @@
+language: "ja-JP"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)